include/myst/thread.h

```diff
@@ -530,4 +551,79 @@ MYST_INLINE void myst_thread_sig_handler_uninstall(
     thread->signal.thread_sig_handler = sig_handler->previous;
 }
 
+MYST_INLINE void myst_thread_queue_push_back(
+    myst_thread_queue_t* queue,
+    myst_thread_t* thread)
+{
+    __myst_thread_queue_push_back(queue, thread, FUTEX_BITSET_MATCH_ANY);
+}
+
+MYST_INLINE void myst_thread_queue_push_back_bitset(
+    myst_thread_queue_t* queue,
+    myst_thread_t* thread,
+    uint32_t bitset)
+{
+    __myst_thread_queue_push_back(queue, thread, bitset);
+}
+
+MYST_INLINE myst_thread_t* myst_thread_queue_pop_front(
+    myst_thread_queue_t* queue)
+{
+    return __myst_thread_queue_pop_front(queue, NULL);
+}
+
+MYST_INLINE myst_thread_t* myst_thread_queue_pop_front_bitset(
+    myst_thread_queue_t* queue,
+    uint32_t* bitset)
+{
+    return __myst_thread_queue_pop_front(queue, bitset);
```

Comment:
Is there a need for a function wrapping this function? Can it be called directly instead?

Reply:
This was part of Mike's work to ensure that the existing flow would remain the same if bitset is not used. @mikbras, is this something that we simplify in a followup PR?